tests/conftest.py

```diff
@@ -103,14 +108,69 @@ def init_teams(g):
         ('identifier3', 'name3'),
         ('identifier4', 'name4'),
     ]
+    # Add teams.
+    created_teams = []
+    for team in init_teams:
+        vteam = g.addV('Team') \
+            .property(T.id, str(uuid.uuid4())) \
+            .property(Cardinality.single, 'identifier', team[0]) \
+            .property(Cardinality.single, 'name', team[1]) \
+            .sideEffect(
+                __.addE("universe_of")
+                .from_(
+                    __.V()
+                    .hasLabel('Universe')
+                    .has('namespace', CURRENT_UNIVERSE.namespace)
+                    .has('version', CURRENT_UNIVERSE.version.int_version)
+                )
+            ) \
+            .elementMap() \
+            .next()
+        created_teams.append(DbTeam.from_vteam(vteam))
+
+    created_teams.sort(key=lambda x: x.vid)
 
+    yield created_teams
+
+
+@pytest.fixture
+def old_universe():
+    """Creates a universe with version 0.0.1 so it can be used in tests as a
+    universe older than the current one.``."""
+    yield Universe(
+        UniverseVersion.from_int_version(
+          CURRENT_UNIVERSE.version.int_version + 1
+        )
+    )
+
+
+@pytest.fixture
+def init_old_universe_teams(g, old_universe):
+    """Creates an initial set of teams and yields a ``DbTeam`` list. They can
+    be used to test the ``InventoryClient``."""
+    init_teams = [
+        ('identifier0', 'name0'),
+        ('identifier1', 'name1'),
+        ('identifier2', 'name2'),
+        ('identifier3', 'name3'),
+    ]
+    g.ensure_universe(universe=old_universe).next()
```

Comment:
I would avoid using the DSL for setting the initial test data.

Reply:
Yeah makes sense, fixed.